lib/ex_sdp/attribute.ex

```diff
@@ -32,36 +38,38 @@ defmodule ExSDP.Attribute do
   `opts` is a keyword list that can contain some information for parsers.
 
   Unknown attributes keys are returned as strings, known ones as atoms.
+  Values for keys `:maxptime`, `:ptime` and `:quality` are converted into `Integer`.
```

Comment:
Maybe let's specify attributes value types along with keys? Can be done in another PR